src/main/commandServer/__tests__/httpCommandServer.spec.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { execFileSync } from 'child_process';
+
+import { HttpCommandServer } from '../httpCommandServer';
+import { spawnFile } from '@/utils/childProcess';
+
+describe(HttpCommandServer, () => {
+  let itWindows = it;
+  let itNonWindows = it;
+  let rdctlPath = path.join('resources', os.platform(), 'bin', 'rdctl');
+
+  if (os.platform().startsWith('win')) {
+    rdctlPath += '.exe';
+    itNonWindows = it.skip;
+    // Don't run the error-message test if the rancher-desktop WSL is running
+    if (execFileSync('wsl', ['--list', '--quiet'], { stdio: 'pipe', encoding: 'utf16le' }).match(/^rancher-desktop$/m) &&
+      execFileSync('wsl', ['--list', '--verbose'], { stdio: 'pipe', encoding: 'utf16le' }).match(/^\s*rancher-desktop\s+Running/m)) {
+      itWindows = it.skip;
```

Comment:
It's surprising that `itWindows` sometimes gets skipped on Windows, isn't it?

Reply:
Not right now -- maybe I can use your `mock-wsl` to force these tests to work as expected under both conditions (`rancher-desktop` created and running, or not), but for now the tests operate under whatever conditions are in effect. So if `rdctl shell` will run, there's no point running the test that handles when it fails.

And this test just handles a particular error message, the one that's issued when `rdctl shell` cannot be run, not others.